config/example/config.txt

```diff
@@ -518,4 +518,10 @@ SQLITE_FEEDBACK_MIN_AGE 7
 #DISABLE_CID_WARN_POPUP
 
 ## Comment this out if you don't want to use the 'nightshift lighting' subsystem to adjust lights based on ingame time
-ENABLE_NIGHT_SHIFTS
\ No newline at end of file
+ENABLE_NIGHT_SHIFTS
+## OOC/LOOC control ##
+
+# Uncomment to allow links of the following kinds. #
+# ALLOW_BYOND_LINKS
+# ALLOW_DISCORD_LINKS
+ALLOW_URL_LINKS

```

Comment:
Didn't you make a comment about having ALLOW_URL_LINKS disabled by default? This example/config.txt enables it by default if I'm not wrong.

Reply:
It's the other way around. If ALLOW_URL_LINKS = 1 (this config here makes it 1), it allows URLs to be enabled. See here:

> if((findtext(msg, "http://") || findtext(msg, "https://")) && !config.allow_url_links)

Reply:
Ah, the OR operator. Fair enough, my mistake.